base/FPMDaemon.php

```diff
@@ -0,0 +1,61 @@
+<?hh
+/*
+ *  Copyright (c) 2016, Intel Corporation.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+final class FPMDaemon extends PHPEngine {
+  private PerfTarget $target;
+
+  public function __construct(private PerfOptions $options) {
+    $this->target = $options->getTarget();
+    parent::__construct((string) $options->fpm);
+
+    // TOOD nice to have: check whether opcache is loaded
```

Comment:
Can you make a request to https://github.com/hhvm/oss-performance/blob/master/base/php-src_config_check.php ?